shared/util/tf/testdata/outputs.tf

```diff
@@ -0,0 +1,18 @@
+output "dashboard" {
+  sensitive = false
+  value     = module.stack.dashboard
+}
+output "service_ecrs" {
+  sensitive = false
+  value     = module.stack.service_ecrs
+}
+output "service_endpoints" {
```

Comment:
```suggestion
output "service_urls" {
```

Reply:
We use service_urls, not service_endpoints in the Happy CLI code

Reply:
https://github.com/chanzuckerberg/happy/blob/45b27099d6504d16f58789e81bca7d7ef1c7e2b0/shared/workspace_repo/workspace.go#L667

Reply:
We do, but the module doesn't have an output of `service_urls`. https://github.com/chanzuckerberg/happy/blob/45b27099d6504d16f58789e81bca7d7ef1c7e2b0/terraform/modules/happy-stack-eks/outputs.tf#LL1C1-L1C1

Reply:
Best I can do is add support for both `service_urls` (manually created HCL) and `service_endpoints` (automatically generated) outputs.